migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -714,8 +714,8 @@ VALUES
   ('NCL', '', 'templates/now/Acknowledgment Letter.docx', true, 'system-mds', 'system-mds'),
   ('NPE', '', 'templates/now/Permit Enclosed Letter.docx', true, 'system-mds', 'system-mds'),
   ('NTR', '[]', 'templates/now/Notice of Work Form.docx', true, 'system-mds', 'system-mds'),
-  ('PMT', '', 'templates/permit/Permit.docx', true, 'system-mds', 'system-mds'),
-  ('PMA', '', 'templates/permit/Permit.docx', true, 'system-mds', 'system-mds')
+  ('PMT', '[]', 'templates/permit/Permit.docx', true, 'system-mds', 'system-mds'),
```

Comment:
I'm pretty sure we pushed this and the below line into development a couple of weeks ago to fix a migration issue so I'm not sure why this is a "new change" all of the sudden.